packages/repository/src/__tests__/unit/repositories/relations-helpers/include-related-models.unit.ts

```diff
@@ -39,14 +39,38 @@ describe('includeRelatedModels', () => {
     expect(result).to.eql([category]);
   });
 
-  it('throws error if the target repository does not have the registered resolver', async () => {
-    const category = await categoryRepo.create({name: 'category 1'});
-    await expect(
-      includeRelatedModels(categoryRepo, [category], [{relation: 'products'}]),
-    ).to.be.rejectedWith(
-      /Invalid "filter.include" entries: {"relation":"products"}/,
-    );
-  });
+  context(
+    'throws error if the target repository does not have registered resolvers',
+    () => {
+      it('the error message reports the invalid entry', async () => {
+        const category = await categoryRepo.create({name: 'category 1'});
+        await expect(
```

Comment:
```
1) includeRelatedModels

       throws error if the target repository does not have registered resolvers

         the error message reports the invalid entry:

     AssertionError: expected [Promise] to be rejected

      at Assertion.fail (packages/testlab/node_modules/should/as-function.js:275:17)

      at /home/travis/build/strongloop/loopback-next/packages/testlab/node_modules/should/as-function.js:1747:16

      at runMicrotasks (<anonymous>)

      at processTicksAndRejections (internal/process/task_queues.js:93:5)

      at Context.<anonymous> (packages/repository/src/__tests__/unit/repositories/relations-helpers/include-related-models.unit.ts:47:9)

  2) includeRelatedModels

       throws error if the target repository does not have registered resolvers

         the error statusCode should be 400:

     TypeError: Cannot read property 'statusCode' of undefined

      at Context.<anonymous> (packages/repository/src/__tests__/unit/repositories/relations-helpers/include-related-models.unit.ts:69:22)

      at runMicrotasks (<anonymous>)

      at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

@agnes512 The travis tests are failing, could you take a look of the reason?

Reply:
Thank you @jannyHou. Fixed.